data/weapons/weapon_data.csv

```diff
@@ -231,20 +231,20 @@ Grom Gatling Gun,diableavionics_strifegun,,,,550,,65,,,30,,10,6.923,10,FRAGMENTA
-Grave LAC (slow),diableavionics_blizzaiaGraveac,1,,750,500,,50,,10,30,6,9,1,3,KINETIC,,,0,0.4,,,5,10,0.5,3,,900,,,100,,,"FIRE_WHEN_INEFFICIENT, SYSTEM","kinetic8, SR",,Diable Avionics,,Anti Shield,,,,,,,,,,
-Cicada Grenade Launcher,diableavionics_blizzaiaCicada,,,,500,,500,,,30,,,,,HIGH_EXPLOSIVE,,,0,7.4,4,0.2,5,10,3,6,,350,450,3.5,300,-0.5,,"FIRE_WHEN_INEFFICIENT,SYSTEM, DO_NOT_CONSERVE",,,,,,,,,,,,,,,
+Grave LAC (slow),diableavionics_blizzaiaGraveac,1,,750,500,,70,,10,30,6,9,1,3,KINETIC,,,0,0.4,,,5,10,0.5,3,,900,,,100,,,"FIRE_WHEN_INEFFICIENT, SYSTEM","kinetic8, SR",,Diable Avionics,,Anti Shield,,,,,,,,,,
+Cicada Grenade Launcher,diableavionics_blizzaiaCicada,,,,500,,500,,,30,,,,,HIGH_EXPLOSIVE,,,0,7.4,5,0.2,5,10,3,6,,350,450,3.5,300,-0.5,,"FIRE_WHEN_INEFFICIENT,SYSTEM, DO_NOT_CONSERVE",,,,,,,,,,,,,,,
 ,diableavionics_blizzaiaShoulderL,,,,1,1,,,,,,,,,,,,,10,,,,,,,,,,,,,,SYSTEM,,,,,,,,,,,,,,,
 ,diableavionics_blizzaiaShoulderR,,,,1,1,,,,,,,,,,,,,10,,,,,,,,,,,,,,SYSTEM,,,,,,,,,,,,,,,
 ,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
 #RAVEN,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
-Burchel (long delay),diableavionics_ravenhead,,,,400,200,,,,36,9,6,0.75,2,ENERGY,,,0,0,0.48,0.02,,,,,1000000,,,,,,,"PD,PD_ONLY,SYSTEM",,,,,,,,,,,,,,,
-Glowtusk Linear Rifle,diableavionics_ravenright,1,,2250,500,,500,,50,25,11,,,,HIGH_EXPLOSIVE,,,0,2.5,,,3,10,3,1,,850,,,350,,,SYSTEM,,,,,,,,,,,,,,,
-Grave HMC,diableavionics_ravenleft,1,,1750,550,,65,,10,30,11,,,,KINETIC,,,0,0.2,1,,1,10,0.5,3,,700,,,100,,,SYSTEM,,,,,,,,,,,,,,,
+Burchel (long delay),diableavionics_ravenhead,,,,600,,150,,,36,9,9,0.75,2,ENERGY,,,0,0,0.48,0.02,,,,,1000000,,,,,,,"PD,PD_ONLY,SYSTEM",,,,,,,,,,,,,,,
+Glowtusk Linear Rifle,diableavionics_ravenright,1,,2250,600,,500,,50,25,11,,,,HIGH_EXPLOSIVE,,,0,2.5,,,3,10,3,1,,850,,,350,,,SYSTEM,,,,,,,,,,,,,,,
+Grave HMC,diableavionics_ravenleft,1,,1750,650,,65,,10,30,11,,,,KINETIC,,,0,0.2,1,,1,10,0.5,3,,700,,,100,,,SYSTEM,,,,,,,,,,,,,,,
 ,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
 #VALIANT WANZER,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
-ILIS PD,diableavionics_valianthead,,,,400,150,,,,30,,,,,ENERGY,,,0.5,0.5,,,,,,,1600,1600,,,,,,"PD,PD_ONLY,SYSTEM",,,,,,,,,,,,,,,
-Cull,diableavionics_valiantleft,,,,400,,100,,,90,,,,,ENERGY,,,0,.75,,,0,5,0.5,5,,2000,,,100,,,SYSTEM,,,,,,,,,,,,,,,
-Cull,diableavionics_valiantright,,,,400,,100,,,90,,,,,ENERGY,,,0,.75,,,0,5,0.5,5,,2000,,,100,,,SYSTEM,,,,,,,,,,,,,,,
+ILIS PD,diableavionics_valianthead,,,,500,150,,,,30,,,,,ENERGY,,,0.5,0.5,,,,,,,1600,1600,,,,,,"PD,PD_ONLY,SYSTEM",,,,,,,,,,,,,,,
+Cull,diableavionics_valiantleft,,,,500,,100,,,90,,,,,ENERGY,,,0,.75,,,0,5,0.5,5,,2000,,,100,,,"SYSTEM,PD_ALSO",,,,,,,,,,,,,,,
+Cull,diableavionics_valiantright,,,,500,,100,,,90,,,,,ENERGY,,,0,.75,,,0,5,0.5,5,,2000,,,100,,,"SYSTEM,PD_ALSO",,,,,,,,,,,,,,,
```

Comment:
Valiant ILIS PD range set to 500 (from 400).
Valiant Cull range set to 500 (from 400).